design/garbage-collection.md

```diff
@@ -39,18 +77,85 @@ In the initial state, both clients have `"ab"`.
 
 ![garbage-collection-2](media/garbage-collection-2.png)
 
-Client `c1` deletes `"b"`, which is recorded as a change with logical timestamp `3`. The text node of `"b"` can be referenced by remote, so it is only marked as tombstone. And the client `c1` sends change `3` to server through PushPull API and receives as a response that `min_synced_seq` is `2`. Since all clients did not receive the deletion `change 3`, the text node is not purged by garbage collection.
+`Client a` deletes `"b"`, which is recorded as a change with versionVector `{b:1}`. The text node of `"b"` can be referenced by remote, so it is only marked as tombstone. And the `Client a` sends change `3a` to server through PushPull API and receives as a response that `minVersionVector` is `{a:1, b:2}`. Since all clients did not receive the deletion `change 3a`, the text node is not purged by garbage collection.
 
-Meanwhile, client `c2` inserts `"c"` after textnode `"b"`.
+Meanwhile, `client b` inserts `"c"` after textnode `"b"`.
```

Comment:
`client b`'s change is local change, so there are not send change `2b` to server and it changes only in local. Am I understanding this correctly?

Reply:
You're right. client b's change has not been sent in that moment(so it stays as local change).
Only client a's change has been sent.

Reply:
How about explaining local change for unfamiliar member to yorkie?